frontend/src/routes/RequestReview.tsx

```diff
@@ -657,6 +667,80 @@ function DatasourceRequestBox({
       : []),
   ];
 
+  /* eslint-disable @typescript-eslint/no-unsafe-call */
+  const fileHandler = async (connectionId: string) => {
+    try {
+      // Create a handle for the file the user wants to save
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
It pops up the error when running the type checking: `Property 'showSaveFilePicker' does not exist on type 'Window & typeof globalThis`
I added this line for Eslint to ignore the `showSaveFilePicker` property as it seems like the typeScript's type definitions for the file system access API is broken

Reply:
Looks like there is a library that would add the types because typescripts typing is broken (called types/wicg-file-system-access). But I'm also fine with this approach.

Reply:
The type `types/wicg-file-system-access` works and just added it, thank you!